src/python/WMCore/Services/pycurl_manager.py

```diff
@@ -305,6 +363,7 @@ def request(self, url, params, headers=None, verb='GET',
         hbuf.flush()
         return header, data
 
+    @portForward(8443)
     def getdata(self, url, params, headers=None, verb='GET',
```

Comment:
this method does not need a decorator.

Reply:
Actually for this one I am pretty sure we need to decorate it because there are several places outside `pycurl_manager` shich import the `RequestHandlerclas` and make a direct call to this method. Some examples are [1], [2], [3]

[1]
https://github.com/dmwm/WMCore/blob/2b737f326694ae57da72543fb733a1aa7c957177/src/python/WMCore/MicroService/Unified/MSRuleCleaner.py#L569

[2]
https://github.com/dmwm/WMCore/blob/2b737f326694ae57da72543fb733a1aa7c957177/src/python/WMCore/MicroService/Unified/Common.py#L363

[3]
https://github.com/dmwm/WMCore/blob/2b737f326694ae57da72543fb733a1aa7c957177/src/python/WMCore/MicroService/Tools/PycurlRucio.py#L92

Reply:
Oops sorry it call `request method directly`.
Agreed. Non need for decoration.